### ISSUE TITLE
How can I disable ConfigCat logging to Xcode debug console?

### ISSUE THREAD

**Issue by hlung** [configcat/swift-sdk#9]
I see it is using OSLog() for most of the logging, which is good. But it logs too many things like `Evaluating rule:` or `Returning User object: {...}`, which clutters up our console log. Is there a way to disable logging to Xcode debug console?

**Comment by z4kn4fein** [comment]
Hey,

Thanks for opening this issue!
I just noticed that for those logs we are using the `.default` logger instead of the pre-configured one with the proper subsystem, so I have to tell the log filtering by level (with the command `log config --mode "level:<level>" --subsystem ""`) is not working right now. 
In the next release, we'll fix this by adding a `loglevel` configuration option.
Is this ok for you?

**Comment by hlung** [comment]
That would be great. The `log config` command is new to me. Thanks for the information :)

**Comment by hlung** [comment]
btw, does `log config` command affects the Xcode debug console logging?

**Comment by z4kn4fein** [comment]
Yep, as I saw, it affects the Xcode debug console too. 
I prepared the next release package with the new logging configuration, I expect it to be released tomorrow.

**Comment by z4kn4fein** [comment]
It's released with v7.1.0, you can now manage the level with the `logLevel` init param:
```
let client = ConfigCatClient(sdkKey: "#YOUR-SDK-KEY#", logLevel: .info)
```
The available options are: 
- `.debug`
- `.info`
- `.warning`
- `.error`
- `.nolog`

With `.nolog` you can turn the logging completely off.

Let me know if you have further questions!
